kimchi/js/bindings.js

```diff
@@ -46,6 +46,22 @@ var caml_bigint_256_of_decimal_string = function (s) {
   );
 };
 
+// Provides: caml_create_rust_empty_vector
+var caml_create_rust_empty_vector = function () {
```

Comment:
This name is misleading.

Reply:
See https://github.com/o1-labs/snarkyjs-bindings/pull/79#discussion_r1282461675